tmt/steps/provision/base.py

```diff
@@ -32,9 +32,25 @@ def copy_from_guest(self, target):
 
             arg: "/var/log/journal.log"
                => f"{provision_dir}/copy/var/log/journal.log
-
         """
-        pass
+        beg = f"[[ -d '{target}' ]]"
+        end = 'exit 0; set -xe; '
+
+        isdir = f"{beg} || {end}"
+        isntdir = f"{beg} && {end}"
+
+        def cmd_mkcp(self, tgt_dir, tgt):
```

Comment:
Please do not make it nested. We've agreed on that with @psss and @thrix; we could possibly move it into common / utils.

Reply:
Un-nested it.